### ISSUE TITLE
[Fix] Portfolio is not red-weak colorblind friendly

### ISSUE THREAD

**Issue by pakoito** [manifoldmarkets/manifold#1209]
![2022-11-18 18_56_33-Pacc (@pakoito) _ Manifold Markets — Mozilla Firefox](https://user-images.githubusercontent.com/514871/202781999-8674de5c-1570-4696-8dd3-a1c1cfe5263a.png)

The page https://manifold.markets/USER?tab=portfolio has colors for investments that are YES (green) or NO (red). As a red-weak colorblind person, it took me a while to realise those were color-coded.

It would be great if the differentiation between the colors was better tuned for colorblind users, maybe with a different shade of red and green, a toggle for other colors, or some other visual indicator.

Market: https://manifold.markets/pakoito/will-manifold-markets-make-the-port

**Comment by sipec** [comment]
yes, I guess the - and + are pretty small :/

curious: does the green/red for YES and NO look similar to you, and does that make it hard to distinguish the UI state for trades or resolutions?

**Comment by pakoito** [comment]
Yeah, I would separate the background behavior from the font color too. Use one of them for whether you have YES or NO, and the other for above or below 0.

The colors for YES/NO in general I can tell apart, but they don't have a lot of contrast when put side-by-side.
